cli/command/environment_test.go

```diff
@@ -1,17 +1,17 @@
 package command
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/quintilesims/layer0/common/models"
 	"github.com/quintilesims/layer0/common/testutils"
-	"github.com/urfave/cli"
 )
 
 func TestCreateEnvironment(t *testing.T) {
 	base, ctrl := newTestCommand(t)
 	defer ctrl.Finish()
-	command := NewEnvironmentCommand(base.Command())
+	command := NewEnvironmentCommand(base.CommandBase()).Command()
```

Comment:
Is there a way to reduce this? Seems a little verbose/ reads weird?

Reply:
It does look/read weird but if you look at the source it will make sense.